packages/@react-stately/data/docs/useTreeData.mdx

```diff
@@ -83,10 +88,11 @@ let tree = useTreeData({
   ],
   initialSelectedKeys: ['Sam', 'Kangaroo'],
   getKey: item => item.name,
-  getChildren: item => item.items
+  getChildren: item => item.items || []
 });
 
 <ListBox
+  aria-label="List organisms"
   items={tree.items}
   selectedKeys={tree.selectedKeys}
   onSelectionChange={tree.setSelectedKeys}>
```

Comment:
Theoretically this is also a breaking change to the types, since people would now have to account for 'all'

Reply:
I think we should punt on supporting `all` for now unless we are comfortable bumping a major since this is a breaking change. I actually ran into something similar for useTreeState where it doesn't support `all` either and we decided to punt (and potentially create a new hook?): https://github.com/adobe/react-spectrum/pull/5756#pullrequestreview-1912741922

I imagine useTreeData will need to be updated pretty heavily in the future when we get to DnD for tree like structures (I ran into a lot of issues when using it with Table sections in https://github.com/adobe/react-spectrum/pull/4444/files), should we wait till then or do you think bumping a major now would be easiest?